t/40-script_openqa-clone-custom-git-refspec.t

```diff
@@ -103,15 +103,15 @@ $expected_re = qr/${expected}/s;
 test_once $args, $expected_re, 'clone-job command with multiple URLs in PR and job URL';
 
 $ENV{curl_github} = qq{echo -e '{"head": {"label": "user:my/branch"}, "body": "Lorem ipsum"}'; true};
-my $needles = 'my/distri/products/sle/needles';
+my $needles = 'products/sle/needles';
```

Comment:
I have to look at it again. I just made tests pass

Reply:
ok. not sure but it looks good to me. I push a commit changing the variable name, hoping that this reflects better to the code and the expected results.

Reply:
with the previous code, the `my/` was making the test fail. `distri` was changed as I had modified the expected variables IIRC